### PR TITLE
Update to Vega 5

### PR DESCRIPTION
**Pull request by domoritz**
Fixes https://github.com/jupyterlab/jupyterlab/issues/6128

I moved the existing extension package to https://github.com/jupyterlab/jupyter-renderers/pull/184

### CODE REVIEW

**Comment by domoritz** [comment]
I believe things are correct but we need another npm release of the new package I added. Can someone help me with that?

**Comment by domoritz** [comment]
I'm lost here. How do I update the yarn lockfile correctly?

File: packages/vega5-extension/package.json [hunk 1]
```diff
@@ -33,10 +33,11 @@
     "@jupyterlab/rendermime-interfaces": "^1.3.0-alpha.3",
     "@phosphor/coreutils": "^1.3.0",
     "@phosphor/widgets": "^1.6.0",
-    "vega-embed": "3.18.2"
+    "vega": "^5.3.1",
+    "vega-embed": "^4.0.0",
+    "vega-lite": "^3.0.0"
```
Comment (ian-r-rose) [inline packages/vega5-extension/package.json]: @domoritz Do these need to be specified here for any reason, or are the `vega` and `vega-lite` pulled in by `vega-embed` sufficient?

Comment (domoritz) [reply packages/vega5-extension/package.json]: Vega-Embed now only has peer dependencies. I think it's much better for you to be able to specify the versions you want regardless of the Vega-Embed version.

Comment (ian-r-rose) [reply packages/vega5-extension/package.json]: Cool, thanks.

Comment (domoritz) [reply packages/vega5-extension/package.json]: Discussion is here: https://github.com/vega/vega-embed/issues/136

**Pull request merged**